### PR TITLE
chip-device-ctrl: add ability to configure reporting (subscribe)

### PR DESCRIPTION
**Pull request by markus-becker-tridonic-com**
Add ability to configure attribute reporting to chip-device-ctrl.

Currently aligned with SiLabs codebase, not the CHIP spec (8.5.2. Subscribe Request Action).

### CODE REVIEW

File: src/controller/python/chip-device-ctrl.py [hunk 1]
```diff
@@ -179,6 +180,7 @@ def __init__(self, rendezvousAddr=None, controllerNodeId=0, bluetoothAdapter=Non
         "resolve",
         "zcl",
         "zclread",
+        "zclconfigure",
```
Comment (Damian-Nordic) [inline src/controller/python/chip-device-ctrl.py:183]: Would `zclsubscribe` be more meaningful?

**Review by Damian-Nordic — CHANGES_REQUESTED** [review]
Thanks for the change! However, it seems like you forgot to include the auto-generated files.

File: src/controller/python/chip-device-ctrl.py (hunk 1 above)
Comment (markus-becker-tridonic-com) [reply src/controller/python/chip-device-ctrl.py:183]: when it is aligned towards CHIP data model, yes. for the moment it was aligned with SiLabs ZCL codebase, which uses the term 'configure reporting'.

Comment (Damian-Nordic) [reply src/controller/python/chip-device-ctrl.py:183]: ok!

**Comment by markus-becker-tridonic-com** [comment]
> Thanks for the change! However, it seems like you forgot to include the auto-generated files.

I can add them in an additional commit. For reviewing it's a bit easier without.

**Review by msandstedt — APPROVED** [review]

**Review by Damian-Nordic — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Comment by markus-becker-tridonic-com** [comment]
@woody-apple the generated files are missing in this PR as discussed above. will you trigger a regeneration?

**Pull request merged**